src/fides/api/service/connectors/scylla_connector.py

```diff
@@ -97,7 +147,21 @@ def mask_data(
         request_task: RequestTask,
         rows: List[Row],
     ) -> int:
-        """Execute a masking request - not yet implemented"""
+        """Execute a masking request"""
+        query_config = self.query_config(node)
+        update_ct = 0
+        client = self.client()
+        for row in rows:
+            generated_update_stmt = query_config.generate_update_stmt(
+                row, policy, privacy_request
+            )
+            if generated_update_stmt is not None:
+                statement, params = generated_update_stmt
+                with client.connect() as connection:
+                    self.set_schema(connection)
+                    connection.execute(statement, parameters=params)
+                    # FIXME: scylla doesn't return the number of updated rows -- unsure if this is important?
```

Comment:
We use the number of updated rows in `src/fides/api/task/graph_task.py` , line 650, in the call to 
```py
self.resources.cache_erasure(
            self.key.value, output
 )
```

not sure if there are any implications to returning None / 0 / some other meaningless value